pages/contact.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react';
+import NavbarBootstrap from '../components/NavbarBootstrap';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Container from 'react-bootstrap/Container';
+
+class Contact extends Component {
```

Comment:
can we make this a stateless/functional component instead of a class-based one? it will be better since we aren't using state or lifecycle methods from within this component so we don't need it to be a class-based component

Reply:
```suggestion
function Contact (props) {
   return (
       <some JSX>
   );
}
```